### PR TITLE
GUI-1539 on AWS, display only the same network elastic IPs for selections on the associate IP dialog

### PR DESCRIPTION
**Pull request by kyolee310**
https://eucalyptus.atlassian.net/browse/GUI-1539

### CODE REVIEW

File: eucaconsole/views/instances.py [hunk 1]
```diff
@@ -326,7 +332,9 @@ def instances_disassociate_ip_address(self):
                 msg = _(u'Successfully disassociated the IP from the instance.')
                 self.request.session.flash(msg, queue=Notification.SUCCESS)
             return HTTPFound(location=self.location)
-        return self.render_dict
+        msg = _(u'Failed to disassociate the IP to the instance.')
```
Comment (kamalgill) [inline eucaconsole/views/instances.py]: @kyolee310: The message should probably be `Failed to disassociate the IP address from the instance`, but please confirm with @JMoLo

File: eucaconsole/static/js/pages/instances.js [hunk 2]
```diff
@@ -52,6 +60,13 @@ angular.module('InstancesPage', ['LandingPage', 'EucaConsoleUtils'])
             $scope.publicDNS = instance.public_dns_name;
             $scope.platform = instance.platform;
             $scope.ipAddress = instance.ip_address;
+            if (action === 'associate-ip-to') {
+                $scope.adjustIPAddressOptions(instance);
```
Comment (kamalgill) [inline eucaconsole/static/js/pages/instances.js]: @kyolee310: If we only need to call the adjustIPAddressOptions() method on AWS, we should wrap that with an AWS check, right?

Comment (kyolee310) [inline eucaconsole/static/js/pages/instances.js]: The function does not only filter the elastic IPs by the network type, but also removes the elastic IPs that are already associated with instances.

File: eucaconsole/views/instances.py (hunk 1 above)
Comment (kyolee310) [inline eucaconsole/views/instances.py]: @JMoLo let us know what the error message should be here.

Comment (JMoLo) [inline eucaconsole/views/instances.py]: The one @kamalgill suggested is fine

Comment (kyolee310) [inline eucaconsole/views/instances.py]: The error messages have been updated.

**Comment by kyolee310** [comment]
retest please

**Comment by kyolee310** [comment]
@aliceh could you restart this test please?

**Comment by aliceh** [comment]
retest this please

**Comment by aliceh** [comment]
@kyolee310 the ip-operations sequence failed at the same place as the previous one. Namely, when associate IP dialog is called from instance detail page then the drop-down is clicked, the IP to be associated  is not found on the list. I tested the same with console from master and it worked.

**Comment by kamalgill** [comment]
Jenkins failure is the problematic `disassociate-ip-from-instance-on-ip-lp` test, which has a random failure with the notifications placeholder.  Safe to ignore, so merging this PR...

**Pull request merged**